src/InputToolbar.js

```diff
@@ -82,7 +82,10 @@ export default class InputToolbar extends React.Component {
   render() {
     return (
       <View
-        style={[styles.container, this.props.containerStyle, { position: this.state.position }]}
+        style={[styles.container, this.props.containerStyle, {
+          position: this.state.position,
+          width: Dimensions.get('window').width,
```

Comment:
wouldn't this calculate the width every time the InputTool rendered? I guess is better to calculate it when the component willMount and when the orientation changes